src/main/java/com/cgi/ipsen5/Controller/UserController.java

```diff
@@ -43,6 +47,19 @@ public ApiResponse<UserResponseDTO> getUserById(@PathVariable UUID id) {
         }
     }
 
+    @GetMapping(path = "/{id}/get-favorite-colleagues")
+    public ApiResponse<List<UserFavoriteColleaguesDTO>> getFavoriteColleaguesByUserId(@PathVariable UUID id){
+        List<UserFavoriteColleaguesDTO> favorites;
+        favorites = this.favoriteColleagueService.getFavoriteColleaguesFromEmployee(id);
+        return new ApiResponse<>(favorites);
+    }
+
+    @PostMapping(path = "/{id}/add-favorite-colleague")
+    public void addFavoriteColleague(@PathVariable UUID id,
+                                     @RequestBody UserFavoriteColleaguesDTO favoriteColleaguesDTO){
+        this.favoriteColleagueService.addFavoriteColleague(id, favoriteColleaguesDTO.getIdOfFavorite());
+    }
```

Comment:
Zou het handiger zijn om te kijken obv welke persoon is ingelogd welke collega's favoriet zijn?

Reply:
Dat gebeurt door de ID van de ingelogde user in url mee te geven :)